client/packages/host/src/api/hooks/index.ts

```diff
@@ -1,7 +1,13 @@
 import { Sync } from './sync';
 import { Utils } from './utils';
+import { Settings } from './settings';
 
 export const useHost = {
+  settings: {
+    displaySettings: Settings.useDisplaySettings,
+    syncSettings: Settings.useSyncSettings,
```

Comment:
no good reason for moving the syncSettings, and I'm having second thoughts about it now..